packages/account/src/BiconomySmartAccount.ts

```diff
@@ -89,41 +89,47 @@ export class BiconomySmartAccount extends SmartAccount implements IBiconomySmart
 
   private isInitialized(): boolean{
     if (!this.isInited)
-    throw new Error('BiconomySmartAccount is not initialized. Please call BiconomySmartAccount.init() before interacting with any other function')
+    throw new Error('BiconomySmartAccount is not initialized. Please call init() on BiconomySmartAccount before interacting with any other function')
     return true
   }
 
   private setProxyContractState(){
+    if ( !BICONOMY_IMPLEMENTATION_ADDRESSES[this.smartAccountInfo.implementationAddress] )
+    throw new Error('Could not find attach implementation address again your smart account. Please generate support ticket for further investegation.')
```

Comment:
Typo in the message, it should be

.. **attached** implementation address **against** your smart account.  
not 
... **attach** implementation address **again** your smart account.

Last message should be
**Please raise an issue on https://github.com/bcnmy/biconomy-client-sdk for further investigation.**

Same goes for other messages.

Reply:
resolved

Reply:
**attach** still not changed to **attached**

Reply:
resolved